torchx/lib/torchx/backend.ex

```diff
@@ -979,6 +979,43 @@ defmodule Torchx.Backend do
     |> to_nx(out)
   end
 
+  @impl true
+  def window_max(out, tensor, window_dims_tuple, opts) do
+    if Enum.any?(opts[:padding], fn conf -> conf |> Tuple.to_list() |> Enum.any?(&(&1 != 0)) end) do
+      raise ArgumentError, "padding unsupported"
+    end
+
+    strides = opts[:strides]
+
+    if tuple_size(window_dims_tuple) > 3 do
+      raise ArgumentError, "cannot have more than 3 dimensions for the window"
+    end
+
+    window_dilations = opts[:window_dilations]
```

Comment:
suggest to move fetch opts into the beginning of the function to make it easy to understand what is supported opts.